src/Plugin/StrawberryRunnersPostProcessorPluginBase.php

```diff
@@ -194,6 +194,43 @@ protected function kill($pid) {
     return strstr(PHP_OS, 'WIN') ? exec("taskkill /F /T /PID $pid") : exec("kill -9 $pid");
   }
 
+  /**
+   * Replace the last occurrence of a given value in the string.
+   * @see https://github.com/phannaly/laravel-helpers/blob/v1.0.3/src/String.php#L331
+   *
+   * @param  string  $search
+   * @param  string  $replace
+   * @param  string  $subject
+   * @return string
+   */
+  public function str_replace_last(string $search, string $replace, string $subject) {
+    $position = strrpos($subject, $search);
+    if ($position !== false) {
+      return substr_replace($subject, $replace, $position, strlen($search));
+    }
 
+    return $subject;
+  }
+
+  /**
+   * Replace the first occurrence of a given value in the string.
+   * @see https://github.com/phannaly/laravel-helpers/blob/v1.0.3/src/String.php#L308
+   *
+   * @param  string  $search
+   * @param  string  $replace
+   * @param  string  $subject
+   * @return string
+   */
+   public function str_replace_first(string $search, string $replace, string $subject) {
```

Comment:
I think we should keep the camelCase notation

Reply:
Make sense. Updating